pay-api/src/pay_api/models/eft_refund.py

```diff
@@ -0,0 +1,64 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Model to handle EFT file processing."""
+from datetime import datetime
+
+from sqlalchemy import ForeignKey
+
+from .base_model import BaseModel
+from .db import db
+
+
+class EFTRefund(BaseModel):  # pylint: disable=too-many-instance-attributes
+    """This class manages the file data for EFT transactions."""
```

Comment:
file data for EFT transactions? Does it? I thought this was just for EFT Refunds?